README.md

```diff
@@ -153,3 +155,41 @@ class CategoryAdmin
 {
 }
 ```
+
+Example admin extensions:
+
+```php
+<?php
+
+namespace App\Entity;
+
+use KunicMarko\SonataAutoConfigureBundle\Annotation as Sonata;
+use Sonata\AdminBundle\Admin\AbstractAdminExtension;
+
+/**
+ * @Sonata\AdminExtensionOptions(
+ *     global=true
+ * )
+ */
+class GlobalExtension extends AbstractAdminExtension
+{
+}
+```
+
+```php
+<?php
+
+namespace App\Entity;
+
+use KunicMarko\SonataAutoConfigureBundle\Annotation as Sonata;
+use Sonata\AdminBundle\Admin\AbstractAdminExtension;
+
+/**
+ * @Sonata\AdminExtensionOptions(
+ *     target="app.admin.category"
```

Comment:
can't this be multiple values?

Reply:
@gremo not sure if you saw the comment

Reply:
https://symfony.com/doc/2.x/bundles/SonataAdminBundle/reference/extensions.html#configuration

Reply:
I guess we should add our compiler pass to process multiple annotations or just make `target` an array and do it like that

Reply:
Which solution do you prefer? I like the second one.

Reply:
I think it will get crowded if we allow multiple annotations, you can end up with 5 annotations just to set different targets, I would rather see target being array field. What do you think?

Reply:
The same. I'll update this PR.